guides/lambda-configuration.md

```diff
@@ -3,15 +3,20 @@
 This covers how to configure and enable AppOptics APM for nodejs lambda functions. For
 general configuration options see `configuration.md` in the same directory.
 
-## Prerequisite
+## Prerequisites
+
+If you haven't already, set up the forwarder. This only needs to be done once per region
+for the account.
```

Comment:
Going back to the question you raised at standup -- there will be an "overview" section in the KB docs about our lambda support and a section on the [forwarder](https://docs.google.com/document/d/1s63i3_NZn-H6r7Wyb4ObercjWxL3292dbkCeWJO3dR4/edit#heading=h.34fnvwia9xym) that you can link to from here... as soon as the KB draft is put in staging ;)

Reply:
Got that. The rest of the doc reads OK to you?

Reply:
Yep looks good!